modules/consentManagement.js

```diff
@@ -163,6 +164,7 @@ export function requestBidsHook(config, fn) {
   nextFn = fn;
   haveExited = false;
   let adUnits = config.adUnits || $$PREBID_GLOBAL$$.adUnits;
+  bidsBackHandler = config.bidsBackHandler;
```

Comment:
is this safe? What about concurrent auctions with distinct bidBackHandler functions?

Reply:
Oh i see this is fine. I assumed it was the global config `import { config } from 'src/config';`

You probably should rename this param so it's not confused with the global.

Reply:
Thanks for the feedback, I renamed the variable.